website/www/site/content/en/get-started/wordcount-example.md

```diff
@@ -387,18 +387,20 @@ python -m apache_beam.examples.wordcount --input YOUR_INPUT_FILE --output counts
 {{< /highlight >}}
 
 {{< highlight class="runner-flink-local" >}}
-Currently, running wordcount.py on Flink requires a full download of the Beam source code.
-See https://beam.apache.org/roadmap/portability/#python-on-flink for more information.
+python -m apache_beam.examples.wordcount --input /path/to/inputfile \
+                                         --output /path/to/write/counts \
+                                         --runner FlinkRunner
 {{< /highlight >}}
 
 {{< highlight class="runner-flink-cluster" >}}
-Currently, running wordcount.py on Flink requires a full download of the Beam source code.
-See https://beam.apache.org/documentation/runners/flink/ for more information.
+# Running Beam Python on a distributed Flink cluster requires additional configuration.
```

Comment:
Would it be more simple and clear for user to keep only one tab for every engine (e.g. just "Flink") and explain all additional instructions, like `local` and `cluster` mode, inside this tab?

Reply:
I agree. I will change this once I can figure out what is going wrong with local cluster execution.